modules/end-user-guide/pages/microsoft-visual-studio-code-open-source-ide.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: CONCEPT
+:description: Microsoft Visual Studio Code - Open Source IDE
+:keywords: user-guide, selecting, IDE, select, selecting, editor, ide, vs-code, visual-studio, visual-studio-code, jetbrains, intellij, pycharm, theia, che-theia
+:navtitle: Microsoft Visual Studio Code - Open Source
+:page-aliases:
+
+[id="microsoft-visual-studio-code-open-source-ide"]
+= Microsoft Visual Studio Code - Open Source IDE
+
+The {prod-short} build of link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source] is the default IDE of a new workspace.
+
+You can have extensions automatically installed in this IDE for workspaces, as explained in the next section, from the xref:administration-guide:extensions-for-microsoft-visual-studio-code-open-source.adoc[Open VSX registry].
+
+TIP: You can use the link:https://code.visualstudio.com/api/ux-guidelines/command-palette[Command Palette] to find and run the commands specified in `devfile.yaml`.
+
+[TIP]
+====
+
+You can use *{prod2}* commands by clicking *{prod2}* in the link:https://code.visualstudio.com/api/ux-guidelines/status-bar[Status Bar] or finding them through the link:https://code.visualstudio.com/api/ux-guidelines/command-palette[Command Palette]:
+
+* *{prod2}: Open Dashboard*
+* *{prod2}: Stop Workspace*
+* *{prod2}: Restart Workspace*
+* *{prod2}: Restart Workspace from Local Devfile*
+* *{prod2}: Open Documentation*
+
+====
+
+NOTE: You might see your organization's branding in this IDE if your organization customized it through a branded build.
```

Comment:
Which IDE is the tip referring to? Whichever one the org uses?

Reply:
:) Good question, and the answer is "this IDE" means "VS Code" because the title of this page is "Microsoft Visual Studio Code - Open Source IDE" (this whole page is about it). So instead of writing "Microsoft Visual Studio Code - Open Source IDE" every time when I need to mention VS Code, I become a minimalist and write "this IDE". Though credit must be given to Nick for this idea.